minio/1.9/README.md

```diff
@@ -0,0 +1,121 @@
+
+- Estimated time for completion: less than 5 minutes. 
+- Target audience:
+    - Operators
+    - Application admins
+    - Developers 
+    - Devops Engineers
+- Scope: Learn to install Minio on DC/OS and learn to use it using Minio command line tool `mc`.
+
+**Table of Contents**:
+
+- [Prerequisites](#prerequisites)
+- [Setting up Minio](#setting-up-minio)
+- [Minio Installation using GUI](#minio-installation-using-gui)
+- [Access Minio Browser](#access-minio-browser)
+- [Install Minio Client](#install-minio-client)
+- [Configure Minio Client](#configure-minio-client)
+- [Uninstalling Minio](#uninstalling-minio)
+- [Explore Further](#explore-further)
+
+## Prerequisites
+
+- DC/OS 1.9 or later
+- [Marathon-LB](https://dcos.io/docs/1.9/usage/service-discovery/marathon-lb/usage/) must be installed and running
+- IP address of the public agent(s) where Marathon-LB or an available hostname configured to point to the public agent(s) where Marathon-LB is running.
+
+## Setting up Minio
+
+You can install Minio Universe package using the DC/OS GUI or CLI.
+
+Before starting, identify the IP address or hostname of a public agent where Marathon-LB is running. 
+
+## Minio installation using GUI
+
+- Visit the DC/OS admin page, and click on Universe on the left menu bar. Then click on the Packages tab and search for Minio, click on the ```Install``` button on the right hand side.
+
+![Search & Install Minio](img/search-install.png)
+
+- Then click on ```Advanced Installation```, enter configuration values like the storage and service type you’d like to use with your Minio instance. Finally specify the IP address or hostname of the public agent where Marathon-LB is running under ```networking >> public agent```. Make sure you remove the leading http:// and the trailing / from the IP.
+
+![Public Agent](img/public-agent.png)
+
+- We're ready to install! Click the green ```Review and Install``` button, verify your settings are correct and then click ```Install``` green button.
+
+![Public Agent](img/verify-install.png)
+ 
+- This completes the install process. Once Minio has been deployed, navigate to the IP/hostname you used earlier for Public Agent. You should see the following login page.
+
+## Access Minio Browser
+
+![Minio Browser](img/minio-browser.png)
+
+- We need the access key and secret key from the Minio container logs. Click on ```Services``` and select Minio service in DC/OS admin page. Then go to the ```logs``` tab and copy the accesskey and secretkey.
+
+![Minio Logs](img/minio-logs.png)
+
+- Navigate to services UI and click on Minio and go to the logs section. Copy the _AccessKey_ and _SecretKey_ and use it to log into the browser console.
+
+![Minio Browser](img/minio-access.png)
+
+- Once you have successfully loggedin you should see the following screen. 
+
+![Minio browser](img/minio-home.png)
+
+### Minio installation using CLI
+
+To install Minio package via CLI, type
+
+```sh
+dcos package install minio
```

Comment:
Unfortunately this doesn't work as is, which is probably why we didn't mention it in the 1.8 example:

$ dcos package install minio
This DC/OS Service is currently in preview. There may be bugs, incomplete features, incorrect documentation, or other discrepancies. Experimental packages should never be used in production!
Continue installing? [yes/no] yes
Error: Options JSON failed validation
Path: networking

Please create a JSON file with the appropriate options, and pass the /path/to/file as an --options argument.

The problem is that it needs the public agent address, so we should probably either remove this section and not mention the CLI (like the 1.8 example), or provide the marathon.json snippit required to pass along the public agent address for this command to work.